deploy/warehouse/target/data.sql

```diff
@@ -5,8 +5,19 @@ begin;
 
 insert into warehouse.organism (lineage, identifiers, details)
     values
+        ('Adenovirus',              'NCBITAXON => 10535',       null),
         ('Bordetella_pertussis',    'NCBITAXON => 520',         '{"report_to_public_health": true}'),
-        ('Influenza',               null,                       null),
+        ('Chlamydophila_pneumoniae','NCBITAXON => 83558',       null),
+        ('Coronavirus.HKU1',        null,                       null),
+        ('Coronavirus.NL63',        'NCBITAXON => 277944',      null),
+        ('Coronavirus.229E',        'NCBITAXON => 11137',       null),
+        ('Coronavirus.OC43',        null,                       null),
+        ('Enterovirus',             'NCBITAXON => 12059',       null),
+        ('Enterovirus.D.68',        'NCBITAXON => 42789',       null),
+        ('Human_bocavirus',         'NCBITAXON => 329641',      null),
+        ('Human_metapneumovirus',   'NCBITAXON => 162145',      null),
+        ('Human_parechovirus',      null,                       null),
+        ('Influenza',               'NCBITAXON => 11309',       null),
```

Comment:
I'm not sure this is the correct NCBI Taxon ID, [11309](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=11309&lvl=3&lin=f&keep=1&srchmode=1&unlock) might be for a specific unidentified _strain_ of Influenza. I don't think `Influenza` exists as a NCBI Taxon entry.